### PR TITLE
spelling fixes and other updates

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: general/administration/building.md [hunk 1]
```diff
@@ -96,17 +96,17 @@ All package builds begin with these two steps:
...
     * The `-InstallNSSM` flag will automatically pull the stable `nssm` binary appropriate to your architecture (x86/x64 only for now) from [NSSM's Website](https://nssm.cc/) and place it in your Jellyfin directory.
 
-7. (Optional) Use [NSSM](https://nssm.cc/) to configure Jellyfin to run as a service.
+7. (Optional) Use [NSSM](https://nssm.cc) to configure Jellyfin to run as a service.
 
-8. Jellyfin is now available in the default directory (or the directory you chose). Assuming you kept the default directory:
+8. Jellyfin is now available in the default directory, or whichever directory you chose.
 
-    * To start it from a Powershell window, run:
+    * Start it from PowerShell
```
Comment (Artiume) [inline general/administration/building.md:103]:
```suggestion
    * Start it from PowerShell.
```

File: general/administration/building.md [hunk 2]
```diff
@@ -96,17 +96,17 @@ All package builds begin with these two steps:
...
 
         ```powershell
         &"$env:APPDATA\Jellyfin-Server\jellyfin.exe"
         ```
 
-    * To start it from CMD, run:
+    * Start it from CMD
```
Comment (Artiume) [inline general/administration/building.md:109]:
```suggestion
    * Start it from CMD.
```

File: general/quick-start.md [hunk 3]
```diff
@@ -7,7 +7,10 @@ title: Quick Start
 
 1. Install Jellyfin on [your system](xref:admin-installing) with the installation method for your platform.
 
-1. [OPTIONAL from 10.7.0 onwards] [Edit the Jellyfin web configuration](xref:web-config) and adjust the options to fit your desired privacy level; our defaults sacrifice some absolute self-hosting for well-requested features, and if this is concerning please review the documentation and edit accordingly.
+1. Edit the [web configuration](xref:web-config) and adjust the options to fit your desired privacy level.
```
Comment (Artiume) [inline general/quick-start.md:10]: if i disable the YouTube plugin in the web config  can't i bypass this with a standalone web?

**Review by Artiume — APPROVED** [review]

File: general/quick-start.md (hunk 3 above)
Comment (dkanada) [reply general/quick-start.md:10]: Yes, but you can always bypass client features with other clients.

File: toc.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 - uid: quick-start
-  href: general/
+  href: general
```
Comment (dkanada) [inline toc.yml:2]:
```suggestion
  href: general/
```

File: toc.yml [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 - uid: getting-help
 - uid: contrib-index
-  href: general/contributing/
+  href: general/contributing
```
Comment (dkanada) [inline toc.yml:5]:
```suggestion
  href: general/contributing/
```

**Pull request merged**